src/main/steps/index.ts

```diff
@@ -143,7 +102,8 @@ const getStepsWithContent = (sequence: Step[], subDir = ''): StepWithContent[] =
   if (sequence?.length) {
     for (const step of sequence) {
       const { url } = parseUrl(step.url);
-      const stepDir = `${dir}${url.startsWith(subDir) ? url : `${subDir}${url}`}`;
+      const subdirurl = url.startsWith(subDir) ? url : `${subDir}${url}`;
```

Comment:
What is the significance of these lines?

Reply:
Its actually came as a part of sonar  issue . And in the suggestions it showed to make literals with conditions  into two parts  .